### ISSUE TITLE
Can't open /dashboard in browser with version 0.8.0

### ISSUE THREAD

**Issue by rabbitom** [FlowFuse/node-red-dashboard#373]
When started, the console shows error:
```
24 Nov 15:32:46 - [error] [ui-base:a2b17a1c347ee607] TypeError: Cannot convert undefined or null to object       
Error: Circular config node dependency detected: f9e1d74c122b476c
    at Flow.start (C:\Users\rabbi\Projects\HolyIoT\holyiot-gateway\node_modules\@node-red\runtime\lib\flows\Flow.js:226:43)
    at async Object.start [as startFlows] (C:\Users\rabbi\Projects\HolyIoT\holyiot-gateway\node_modules\@node-red\runtime\lib\flows\index.js:398:17)
24 Nov 15:32:46 - [error] [ui-button:6ff66036881ef88f] No group configured
```
It resolves after down grade to version 0.7.2

**Comment by joepavitt** [comment]
Ooh, thanks for flagging. As I'm not able to reproduce, are you able to re-try this please, there _should_ be some more errors above this one detailing what has actually broken?

The circular dependency is Node-RED having tried and tried, but failed to instantiate. There is likely a few instances of this error, but above all of them, there will be a unique "this is the acual problem"

**Comment by joepavitt** [comment]
Noting that it's also reporting:

> 24 Nov 15:32:46 - [error] [ui-button:6ff66036881ef88f] No group configured

**Comment by joepavitt** [comment]
Closed by #435
